lib/v2/screens/profile_screens/citizenship/citizenship_screen.dart

```diff
@@ -0,0 +1,195 @@
+import 'package:seeds/i18n/citizenship.18n.dart';
+
+/// CITIZENSHIP SCREEN
+class CitizenshipScreen extends StatefulWidget {
+  const CitizenshipScreen();
+
+  @override
+  _CitizenshipScreenState createState() => _CitizenshipScreenState();
+}
+
+class _CitizenshipScreenState extends State<CitizenshipScreen> with TickerProviderStateMixin {
+  AnimationController _controller;
+  Animation<double> _timeLineAnimation,
+      _reputationAnimation,
+      _visitorsAnimation,
+      _ageAnimation,
+      _seedsAnimation,
+      _transactionsAnimation,
+      _friendsAnimation;
+  int _timeLine = 0, _reputation = 0, _visitors = 0, _age = 0, _seeds = 0, _transactions = 0, _friends = 0;
+
+  @override
+  void initState() {
+    _controller = AnimationController(duration: const Duration(seconds: 1), vsync: this);
+    _timeLineAnimation = Tween<double>(begin: 0, end: 85).animate(_controller)
+      ..addListener(() {
+        setState(() => _timeLine = _timeLineAnimation.value.toInt());
+      });
+    _reputationAnimation = Tween<double>(begin: 0, end: 50).animate(_controller)
+      ..addListener(() {
+        setState(() => _reputation = _reputationAnimation.value.toInt());
+      });
+    _visitorsAnimation = Tween<double>(begin: 0, end: 1).animate(_controller)
+      ..addListener(() {
+        setState(() => _visitors = _visitorsAnimation.value.toInt());
+      });
+    _ageAnimation = Tween<double>(begin: 0, end: 59).animate(_controller)
+      ..addListener(() {
+        setState(() => _age = _ageAnimation.value.toInt());
+      });
+    _seedsAnimation = Tween<double>(begin: 0, end: 200).animate(_controller)
+      ..addListener(() {
+        setState(() => _seeds = _seedsAnimation.value.toInt());
+      });
+    _transactionsAnimation = Tween<double>(begin: 0, end: 3).animate(_controller)
+      ..addListener(() {
+        setState(() => _transactions = _transactionsAnimation.value.toInt());
+      });
+    _friendsAnimation = Tween<double>(begin: 0, end: 1).animate(_controller)
+      ..addListener(() {
+        setState(() => _friends = _friendsAnimation.value.toInt());
+      });
+    _controller.forward();
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    _controller.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Column(
+                  children: [
+                    const ProfileAvatar(
+                      size: 100,
+                      image: '',
+                      nickname: 'raul',
+                      account: 'raul',
+                    ),
+                    const SizedBox(height: 8.0),
+                    Text(
+                      'Raul',
```

Comment:
same here

Reply:
Because I didn't want to add more logic to avoid making the PR bigger, the fact that I still don't know where the scores are retrieved from for example and if I should retrieve the values ​​from the cache or make a call.